README.md

```diff
@@ -12,78 +16,94 @@ Some paragraphs are considered "internal", and only meant to be used inside othe
 
-- Enable the OpenEuropa Whitelabel Theme and set it as default ```./vendor/bin/drush config-set system.theme default oe_whitelabel_theme```
+Ideally this project should be managed with [Docker](https://www.docker.com/get-docker) and [Docker Compose](https://docs.docker.com/compose/), but this is not a hard requirement.
 
-### Integration with oe_paragraphs
+Check the [composer.json](composer.json) for required PHP version and other dependencies.
 
-In order to have full working integration with paragraphs in your project, you must enable oe_whitelabel_paragraphs module:
+### Add the composer package
 
-```./vendor/bin/drush en oe_whitelabel_paragraphs```
+Add this manually in composer.json, or combine with existing entries:
 
-## Development setup
+```
+    "extra": {
+        "artifacts": {
+            "openeuropa/oe_bootstrap_theme": {
+                "dist": {
+                    "url": "https://github.com/{name}/releases/download/{pretty-version}/{project-name}-{pretty-version}.zip",
+                    "type": "zip"
+                }
+            },
+            "openeuropa/oe_whitelabel": {
+                "dist": {
+                    "url": "https://github.com/{name}/releases/download/{pretty-version}/{project-name}-{pretty-version}.zip",
+                    "type": "zip"
+                }
+            }
+        }
+    }
+```
 
-OpenEuropa Whitelabel theme uses [Webpack](https://webpack.js.org) to compile and bundle SASS and JS.
+If your project is still in development, you may modify composer.json to allow alpha versions, while this package and some of its dependencies are still in their `alpha` phase.
 
-Make sure you have Node and npm installed.
+Review this again when the site goes into production.
 
-You can read a guide on how to install node here: https://docs.npmjs.com/getting-started/installing-node
+```
+    "minimum-stability": "alpha",
+```
 
-If you prefer to use [Yarn](https://yarnpkg.com) instead of npm, install Yarn by following the guide [here](https://yarnpkg.com/docs/install).
+Require with composer:
 
-To install required Node.js dependencies run:
+```bash
+composer require openeuropa/oe_whitelabel
+```
 
+If composer.json does not allow alpha versions, a version string needs to be appended to the package name in the require command, and other dependencies with alpha versions also need to be required with explicit version strings. The version string could be `:^1.0@alpha`, or a specific version like `:^1.0.0-alpha7`.
 
-`npm install` or `yarn install`.
+```bash
+composer require openeuropa/oe_whitelabel:^1.0@alpha openeuropa/oe_bootstrap_theme:^1.0@alpha
+```
 
-To build the final artifacts run:
+Review the installed package versions. Make sure that oe_whitelabel and oe_bootstrap theme are using 1.x or higher.
 
 ```bash
-npm run build
+composer show -i | grep oe_
 ```
```

Comment:
The motivation were situations where me or other people wanted to install and got releases from 0.x, because these are considered stable.
I will keep it around for 2nd round, but I am not fully against removing it.

Reply:
Also not sure what this has to do with the submodules.

Reply:
I think its ok as is, it helps to bring attention to the releases being used.